### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by frgis** [mviewer/mviewer#942] [General]
Bonjour,

nous comptons passer une mise à jour de Mviewer d'ici fin d'année 2024, pouvez-vous nous indiquer si la release actuelle (3.12.1) sera la dernière de 2024 ? Si non, vers quelle date vous pensez sortir la dernière release ?

Merci.
Cordialement.

**Comment by lecault** [comment]
Bonjour,

Je pense que l'on fera une release prochainement (d'ici fin 2024) avec la finalisation de l'intégration des fonctionnalités liées à OGC API.

**Reply by frgis** [reply]
Ok, vous pensez que ce sera avant mi-décembre ou après ?

**Reply by lecault** [reply]
Avant. Si pas de release mi decembre, ça sera en 2025

**Reply by frgis** [reply]
ok, super. Merci pour votre réactivité !
Bonne soirée

**Comment by lecault** [comment]
La dernière version vient de paraître : https://github.com/mviewer/mviewer/releases/tag/v3.13

**Comment by frgis** [comment]
Ok, merci pour l'info !
